.circleci/config.yml

```diff
@@ -62,6 +62,18 @@ commands:
       - store_artifacts:
           path: ~/artifacts
 
+  makesmoke:
```

Comment:
This CI config is setup very differently than our newer projects; this `makesmoke` step was intentionally added in a way that matched the existing config styles. It may be worthwhile to restructure this config in a future PR for consistency.